docs/source/tensorbay_cli/dataset_management.rst

```diff
@@ -1,3 +1,183 @@
 ####################
```

Comment:
Maybe the following commit message is better:
docs: split getting_start_with_CLI.rst into two files